js/backboneAgent/controllers/appViewController.js

```diff
@@ -151,6 +159,47 @@ Modules.set('controllers.appViewController', function() {
 
                 return result;
             }});
+        },
+
+        setupHighlight: function() {
+            this.highlightMask = document.createElement('div');
+            this.highlightMask.style.position = 'absolute';
+            this.highlightMask.style.zIndex = '100000000000';
+            this.highlightMask.style.pointerEvents = 'none';
+            this.highlightMask.style.backgroundColor = 'rgba(55, 161, 243, 0.48)';
+            this.highlightMask.style.webkitFilter = 'grayscale(20%)';
+
+            // unhighlight when the user closes the devtools, etc.
+            this.listenTo(port, 'client:disconnect', this.unhighlightViewElements);
```

Comment:
Should this callback, or the `unhighlightViewElements` method also remove the `this.highlightMask` element from the DOM to avoid polluting it once the inspector is closed?